proc/process.c

```diff
@@ -1068,10 +1068,15 @@ static void *process_putargs(void *stack, char ***argsp, int *count)
 }
 
 
+#define STACK_ARGSZ (STACK_SIZE(char **) + STACK_SIZE(char **) + STACK_SIZE(int) + STACK_SIZE(void *))
```

Comment:
I rather remove this macro, because:
- it's used exactly once, we have a doctrine of avoiding macros is feasible,
- its value is detached from it's meaning - it's usage defines why there are values that there are.